tasks/quality.js

```diff
@@ -46,9 +40,7 @@ module.exports = function(grunt) {
   validate.push('phplint:all');
 
   if (grunt.config.get('config.phpcs')) {
-    var phpcs = grunt.config.get('config.phpcs.dir') || [
-        '<%= config.srcPaths.drupal %>/**/*.css'
```

Comment:
Killed this off to prevent globbing. It seems generally undesirable in most cases to run phpcs on css files anyway, since they are almost always generated.